framework/tests/viewpointtest/VarargsConstructor.java

```diff
@@ -47,13 +54,17 @@ void invokeConstructor(@A Object aObj, @B Object bObj, @Top Object topObj) {
 
     void testAnonymousClass(@A Object aObj, @B Object bObj, @Top Object topObj) {
         Object o =
-                new VarargsConstructor("testStr", new Object()) {
+                // :: warning: (cast.unsafe.constructor.invocation)
+                new @A VarargsConstructor("testStr", new @A Object()) {
                     void foo() {
-                        VarargsConstructor a = new VarargsConstructor("testStr", new Object());
+                        VarargsConstructor a =
+                                // :: warning: (cast.unsafe.constructor.invocation)
+                                new @A VarargsConstructor("testStr", new @A Object());
```

Comment:
Is this change illustrating something about `Lost`? Should there be versions of this test without an annotation? A comment would help understand this test.

Reply:
This change is not about lost, I update this test case because I want those invalid invocation because previous we allow `new @Top` constructor invocation.  More test cases for lost is in `LostNonReflexive.java`